kubernetes.yaml

```diff
@@ -89,6 +89,8 @@ data:
     namespace    = default
     
     max_proc     = 2
+    
+    logging_level = DEBUG
```

Comment:
not by default!

Reply:
please leave out this line, in normal use, one shouldn't need to touch it (INFO is fine)